components/script/dom/node.rs

```diff
@@ -2499,22 +2514,31 @@ impl Node {
             }
         }
 
-        // Step 14. If node has an inclusive descendant that is a slot:
+        // Step 10. If node has an inclusive descendant that is a slot:
         let has_slot_descendant = node
             .traverse_preorder(ShadowIncluding::No)
             .any(|elem| elem.is::<HTMLSlotElement>());
         if has_slot_descendant {
-            // Step 14.1 Run assign slottables for a tree with parent’s root.
+            // Step 10.1 Run assign slottables for a tree with parent’s root.
             parent
                 .GetRootNode(&GetRootNodeOptions::empty())
                 .assign_slottables_for_a_tree();
 
-            // Step 14.2 Run assign slottables for a tree with node.
+            // Step 10.2 Run assign slottables for a tree with node.
             node.assign_slottables_for_a_tree();
         }
 
-        // Step 11. transient registered observers
-        // Step 12.
+        // Step 11. Run the removing steps. It is handled in unbind_from_tree.
+        // TODO: double check if that's correct
+
+        // TODO: Step 12. isParentConnected
+
+        // Step 13-14. Enqueue disconnected callback reactions for custom elements.
+        Self::enqueue_disconnected_cereaction_for_subtree(node);
```

Comment:
Is CEReaction a proper noun from the specification? If not, I would avoid using it in a function name and just spell things out. I just mention this because "cereaction" is pretty tricky for me to read -- it looks very similar to "creation."